README.md

```diff
@@ -1,3 +1,339 @@
 # npecl design
 
-more info as it is written...
+A work in progress design documentation for the "new pecl" iteration.
+
+Basis:
+
+ * [Discussion doc](https://docs.google.com/document/d/1_N0E9xo3jn9aKrIZHIbTYaY5lXw71BpSO6-it4cRpDo/edit)
+ * Internal PHPF dev meet kick-off to review (minutes?)
+
+## Name suggestions
+
+ * npecl
+ * Cucumber/Pickle adjacent:
+   * 🥦 Floret / Broccoli
+   * 🌱 Seedling / Sapling
+   * 🍇 Grape / GraPHPe
+   * 🥒 Enpickle / Pickled
```

Comment:
Another one - gherkin

Reply:
It did cross my mind, but I think we should not use Gherkin, to avoid confusion with [Gherkin](https://cucumber.io/docs/gherkin/) syntax (which is also used in [Behat PHP Gherkin parser](https://github.com/Behat/Gherkin), as mentioned by Derick further down).